topology/zz-bond/cpn.yaml

```diff
@@ -0,0 +1,21 @@
+cpn_nodes:
+    nz-kiwi-t1sw1:
+        cpn_ip: 127.0.0.1
+        role: dist
+        vendor: NoviSwitch
+        model: NS-2128
+    nz-kiwi-t1sw2:
+        cpn_ip: 127.0.0.1
+        role: dist
+        vendor: NoviSwitch
+        model: NS-2128
+    nz-kiwi-t2sw1:
+        cpn_ip: 127.0.0.1
```

Comment:
shouldn't one of these be "wrong" to test for failure?   Doesn't matter much now, but at some point we'll be testing pass/fail conditions.